deploy/test-app/apply/TestAppGateway.yaml

```diff
@@ -78,4 +78,14 @@ spec:
       get:
         x-fabric-employee-access:
           user-list:
-            - notreal
\ No newline at end of file
+            - notreal
+    /static:
```

Comment:
I don't see the schema updates in this PR. Should there not be changes to the `deploy/operator/apply/01_FabricGatewayCRD.yaml` CRD defn?

Reply:
Ah I forgot about them, will do them :+1: